### PR TITLE
AGW: pipelined: QoS: Fix max bandwidth calculation.

### LINKED ISSUE #6289: [agw][pipelined] QoS Enforcement is not working with pyroute2 enabled
### Your Environment

- **Version:** master
- **Affected Component:** Access Gateway
- **Affected Subcomponent:** Pipelined
- **Deployment Environment:**  Vagrant (AGW)

### Describe the Issue

In the default setting of `enable_pyroute2: true` in pipelined.yml file, QoS enforcement is failing. When the setting is changed to `enable_pyroute2: false`, QoS enforcement is working as expected. 

**To Reproduce**
1. Bring up AGW test set up with magma, magma_test and magma_trfserver VMs. 
2. Disable tcp checksum on all three machines via running the command `disable-tcp-checksumming`.
3. Make sure that magma services are running on magma VM and traffic server is started on magma_trfserver VM.
4. Run the integration test with `make integ_test TESTS=s1aptests/test_attach_detach_setsessionrules_tcp_data.py` under directory `magma/lte/gateway/python/integ_tests` on magma_test VM.

Test fails on asserting the achieved rate (that it should be less than the MBR set in the test case). Also check the TC rules and packet marking rules using. Observe that TC rules are wrong.

```
echo 'DUMPING ETH1'
sudo tc -s -d class show dev eth1
sudo tc -s -d filter show dev eth1

echo 'DUMPING ETH2'
sudo tc -s -d class show dev eth2
sudo tc -s -d filter show dev eth2

echo 'Dumping flows'
sudo ~/build/python/bin/pipelined_cli.py debug display_flows | grep 'PKT_MARK'
```

Change the setting in pipelined.yml file and disable pyroute, restart magma services, rerun the test on step 4. Observe that the test passes this time.

**Expected behavior**
Test should pass whether or not the flag `enable_pyroute2` set as true or false.

### PR DESCRIPTION
**Pull request by pshelar**
Pyroute2 API expects max-bandwidth parameter in bytes rather than
bit rate.

Signed-off-by: Pravin B Shelar <pbshelar@fb.com>

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

File: lte/gateway/python/magma/pipelined/qos/tc_ops_pyroute2.py [hunk 1]
```diff
@@ -37,9 +37,10 @@ def __init__(self):
 
     def create_htb(self, iface: str, qid: str, max_bw: int, rate: str,
                    parent_qid: str = None) -> int:
-        LOG.debug("Create HTB iface %s qid %s", iface, qid)
+        LOG.debug("Create HTB iface %s qid %s max_bw %s rate %s", iface, qid, max_bw, rate)
 
         try:
+            max_bw = max_bw / 8
```
Comment (koolzz) [inline lte/gateway/python/magma/pipelined/qos/tc_ops_pyroute2.py:57]: Maybe add a note that api needs bytes

**Review by koolzz — APPROVED** [review]

File: lte/gateway/python/magma/pipelined/qos/tc_ops_pyroute2.py [hunk 2]
```diff
@@ -37,9 +37,23 @@ def __init__(self):
...
+            rate: rate limiting.
+            parent_qid: HTB parent queue.
+
+        Returns:
+            zero on success.
+        """
+
+        LOG.debug("Create HTB iface %s qid %s max_bw %s rate %s", iface, qid, max_bw, rate)
         try:
+            # API needs ceiling in KBytes per sec.
```
Comment (ulaskozat) [inline lte/gateway/python/magma/pipelined/qos/tc_ops_pyroute2.py]: KBytes or Bytes?

**Review by ulaskozat — APPROVED** [review]
There are some legit style guide comments. Otherwise looks good.

File: lte/gateway/python/magma/pipelined/qos/tc_ops_pyroute2.py (hunk 2 above)
Comment (pshelar) [reply lte/gateway/python/magma/pipelined/qos/tc_ops_pyroute2.py]: right, its byte

**Comment by ulaskozat** [comment]
@pshelar this should be also backported to 1.4 right?

**Comment by themarwhal** [comment]
Removed the tag so it can be tracked here : https://github.com/magma/magma/issues/6827

**Pull request merged**